Description: Upstream changes introduced in version 0.4.4-30yavdr0~oneiric
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 vdr-plugin-menuorg (0.4.4-30yavdr0~oneiric) oneiric; urgency=medium
 .
   * fixed "'NULL' was not declared in this scope"
 .
 The person named in the Author field signed this changelog entry.
Author: Gerald Dachs <gda@dachsweb.de>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- vdr-plugin-menuorg-0.4.4.orig/src/CommandMenuNode.cpp
+++ vdr-plugin-menuorg-0.4.4/src/CommandMenuNode.cpp
@@ -21,6 +21,8 @@
  *
  */
 
+#include <cstddef>
+
 #include "CommandMenuNode.h"
 #include <vdr/interface.h>
 #include <vdr/menu.h>
--- vdr-plugin-menuorg-0.4.4.orig/src/MenuOrgPlugin.cpp
+++ vdr-plugin-menuorg-0.4.4/src/MenuOrgPlugin.cpp
@@ -21,6 +21,8 @@
  *
  */
 
+#include <cstddef>
+
 #include <vdr/plugin.h>
 #include <vdr/config.h>
 #include <vdr/tools.h>
--- vdr-plugin-menuorg-0.4.4.orig/src/MenuNode.cpp
+++ vdr-plugin-menuorg-0.4.4/src/MenuNode.cpp
@@ -21,6 +21,8 @@
  *
  */
 
+#include <cstddef>
+
 #include "MenuNode.h"
 #include "SubMenuNode.h"
 
--- vdr-plugin-menuorg-0.4.4.orig/src/OsdItemDefinition.cpp
+++ vdr-plugin-menuorg-0.4.4/src/OsdItemDefinition.cpp
@@ -21,6 +21,8 @@
  *
  */
 
+#include <cstddef>
+
 #include "OsdItemDefinition.h"
 
 OsdItemDefinition::OsdItemDefinition(MenuNode* menuNode, cOsdItem* osdItem, bool isSelected)
--- vdr-plugin-menuorg-0.4.4.orig/src/MenuOrgPlugin.h
+++ vdr-plugin-menuorg-0.4.4/src/MenuOrgPlugin.h
@@ -24,6 +24,8 @@
 #ifndef ___MENUORGPLUGIN_H
 #define ___MENUORGPLUGIN_H
 
+#include <cstddef>
+
 #include <vdr/plugin.h>
 #include <string>
 #include "PluginConfiguration.h"
--- vdr-plugin-menuorg-0.4.4.orig/src/PluginItemDefinition.cpp
+++ vdr-plugin-menuorg-0.4.4/src/PluginItemDefinition.cpp
@@ -21,6 +21,8 @@
  *
  */
 
+#include <cstddef>
+
 #include "PluginItemDefinition.h"
 
 using namespace std;
--- vdr-plugin-menuorg-0.4.4.orig/src/MainMenuItemsProvider.cpp
+++ vdr-plugin-menuorg-0.4.4/src/MainMenuItemsProvider.cpp
@@ -21,6 +21,8 @@
  *
  */
 
+#include <cstddef>
+
 #include "MainMenuItemsProvider.h"
 #include "SubMenuNode.h"
 #include "MenuConfigurationRepository.h"
--- vdr-plugin-menuorg-0.4.4.orig/src/MenuConfigurationRepository.cpp
+++ vdr-plugin-menuorg-0.4.4/src/MenuConfigurationRepository.cpp
@@ -21,6 +21,8 @@
  *
  */
 
+#include <cstddef>
+
 #include "MenuConfigurationRepository.h"
 #include <exception>
 #include <iostream>
--- vdr-plugin-menuorg-0.4.4.orig/src/PluginMenuNode.cpp
+++ vdr-plugin-menuorg-0.4.4/src/PluginMenuNode.cpp
@@ -21,6 +21,8 @@
  *
  */
 
+#include <cstddef>
+
 #include "PluginMenuNode.h"
 #include <vdr/plugin.h>
 #include "ChildLock.h"
--- vdr-plugin-menuorg-0.4.4.orig/src/ChildLock.cpp
+++ vdr-plugin-menuorg-0.4.4/src/ChildLock.cpp
@@ -21,6 +21,8 @@
  *
  */
 
+#include <cstddef>
+
 #include "ChildLock.h"
 #include <vdr/plugin.h>
 #include "IChildLockService.h"
